src/elements/content-uploader/ContentUploader.js

```diff
@@ -746,7 +786,7 @@ class ContentUploader extends Component<Props, State> {
         const { onUpload, useUploadsManager } = this.props;
 
         item.progress = 100;
-        if (!item.error) {
+        if (item.status !== STATUS_ERROR) {
```

Comment:
Why are we changing this to check status? Should it be that we only mark the status as complete if there is no error?